xml/common_intro_feedback_i.xml

```diff
@@ -7,7 +7,7 @@
 
 <!--taroth 080104: included several times, therefore no ID here, all products-->
 <sect1 xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0">
- <title>Feedback</title>
+ <title>Providing Feedback</title>
```

Comment:
How about just "Giving"?

Reply:
Let's see what translations make out of this, but yes, why not. :)